lib/bolt/outputter/human.rb

```diff
@@ -59,10 +59,14 @@ def print_result(node, result)
         if result.instance_of? Bolt::TaskResult
           @stream.puts(indent(2, ::JSON.pretty_generate(result.value)))
         elsif result.instance_of? Bolt::CommandResult
-          @stream.puts(indent(2, "STDOUT:"))
-          @stream.puts(indent(4, result.stdout))
-          @stream.puts(indent(2, "STDERR:"))
-          @stream.puts(indent(4, result.stderr))
+          unless result.stdout.strip.empty?
```

Comment:
Can `result.stdout` or `result.stderr` be nil?

Reply:
Not with the existing transports but I'll set that to be sure.